examples/user_guide/08-Gridded_Datasets.ipynb

```diff
@@ -13,7 +13,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import xarray as xr\n",
```

Comment:
Unfortunately I made a manual error here when dealing with the flakes caught by nbsmoke - `xr` is used. I'll make a new PR (probably alongside doing python 2 lint checks).

Reply:
Thanks!